distribution/docs/src/main/resources/content/_architectures/_catalogFrameworks/catalog-api.adoc

```diff
@@ -101,30 +100,16 @@ Resources may exist either locally or on a remote data store.
 * ${branding} REST
 
 The Query Service Endpoint, the Catalog Framework, and the `CatalogProvider` are key
-components for processing a retrieve product request.
-The Endpoint bundle contains a Web service that exposes the interface to retrieve products, also referred to as Resources.
+components for processing a retrieve resource request.
+The Endpoint bundle contains a Web service that exposes the interface to retrieve resources.
 The Endpoint calls the `CatalogFramework` to execute the operations of its specification.
-The `CatalogFramework` relies on the Sources to execute the actual product retrieval.
-Optional PreResource and PostResource Catalog Plugins may be invoked by the `CatalogFramework` to modify the product retrieval request/response prior to the Catalog Provider processing the request and providing the response.
-It is possible to retrieve products from specific remote Sources by specifying the site name(s) in the request.
+The `CatalogFramework` relies on the Sources to execute the actual resource retrieval.
+Optional `PreResource` and `PostResource` Catalog Plugins may be invoked by the `CatalogFramework` to modify the resource retrieval request/response prior to the Catalog Provider processing the request and providing the response.
+It is possible to retrieve resources from specific remote Sources by specifying the site name(s) in the request.
 
 .Product Caching
-[NOTE]
-====
-Existing ${branding} clients are able to leverage product caching due to the product cache being implemented in the ${branding}.
-Enabling the product cache is an administrator function.
-
 Product Caching is enabled by default.
-====
-
-To configure product caching:
-
-. Navigate to the *${admin-console}*.
-. Select ${ddf-catalog}.
-. Select *Configuration*.
-. Select *Resource Download Settings*.
-
-See <<{reference-prefix}ddf.catalog.resource.download.ReliableResourceDownloadManager,Resource Download Settings configurations>> for all possible configurations.
```

Comment:
❓ Should we include a link to https://github.com/codice/ddf/pull/5608/files#diff-de98e1cea538c46c6ad3f393cd560bd7R1?

Reply:
I'm not sure. Can you explain?